web_services/foxx/brightid5/db.js

```diff
@@ -425,6 +462,7 @@ function addMembership(groupId, key, timestamp) {
   if (groupMembers(groupId).length == group.founders.length) {
     groupsColl.update(group, { isNew: false });
   }
+  updateEligibles(groupId);
```

Comment:
From what I can see in https://github.com/BrightID/BrightID-Node/blob/bbe3dceb71b19ddca1208992658e2209c034bd82/web_services/foxx/brightid5/index.js#L106-L107

`db.userGroups()` and `db.userInvitedGroups()` are going to update the eligible members of the group, making the next line https://github.com/BrightID/BrightID-Node/blob/bbe3dceb71b19ddca1208992658e2209c034bd82/web_services/foxx/brightid5/index.js#L108
redundant. This seems to be the only usage of https://github.com/BrightID/BrightID-Node/blob/bbe3dceb71b19ddca1208992658e2209c034bd82/web_services/foxx/brightid5/db.js#L159
so the call to it and the function itself could be removed, right?

Reply:
> db.userGroups() and db.userInvitedGroups() are going to update the eligible members of the group, making the next line

No, these 2 methods do not update anything in database. They are only reading `groups` and `invitations` of the user.
`updateEligibleGroups` is updating database based on connections created/removed or users joined/left different groups that may result in the current user be eligible to join some groups or not be eligible for.
This is traditional way we updated `eligible_groups` attribute on `users`. We added a new function called `updateEligibles` which is called on every group when someone join that group to search for users who are eligible to join that group and update their `eligible_groups`.

Reply:
You're right.  Because many lines were collapsed in this pull request view, I mistakenly thought that the call to  `updateEligibles()` was part of the `groupToDic()` function, when in fact it's part of the `addMembership()` function